modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -6,51 +6,69 @@ endif::[]
+image::using-munit-coverage-in-anypoint-studio.png["MUnit Coverage tab highlighted in the MUnit section."]
 
-The Overall Coverage value represents the percentage of the Mule application event processors that have been executed by the MUnit test.
+The Overall Coverage value represents the percentage of the Mule application event processors that the MUnit test executes.
 
-You can click the Generate Report button to get the report detailing the flows and event processors contained by the Mule configuration files that were included in the MUnit test.
+You can click on *Generate Report* to get the report that details the flows and event processors in the Mule configuration files that are included in the MUnit test.
 
-image::munit-coverage-report-in-anypoint-studio.png[]
+image::munit-coverage-report-in-anypoint-studio.png["Files included in the report are highlighted."]
 
 [%header,cols="20a,80a"]
 |===
 | Column      | Description
-| *Resource*    | The name of the Mule Configuration file.
-| *Containers* | Number of Flows inside the Mule configuration file.
-| *Weight*      | This field represents how much of your application is contained inside a configuration file. +
+| *Resource*    | The name of the Mule Configuration file
+| *Containers* | Number of flows inside the Mule configuration file
+| *Weight*      | The percentage of your application in a configuration file. +
                 It is calculated as a percentage of the overall number of event processors in the documentation and the
-                amount of event processors contained in the Mule configuration file.
-| *Coverage*    | This field represents the percentage of the event processors inside the Mule Configuration file that
-                were executed in the MUnit test.
+                amount of event processors in the Mule configuration file.
+| *Coverage*    | The percentage of the event processors inside the Mule Configuration file that is executed in the MUnit test
 |===
 
-Additionally you can click in each Resource, to get a more specific and granular report of the Event Processors executed in that resource:
+Additionally, you can click on each *Resource* to get a more specific and granular report of the Event Processors executed in that resource:
 
-image::munit-granular-coverage-report-in-anypoint-studio.png[]
+image::munit-granular-coverage-report-in-anypoint-studio.png["Granular report is highlighted."]
 
 [%header,cols="20a,80a"]
 |===
 | Column              | Description
-| *Name*                | The name of the container.
-| *Type*                | The type of the container.
-| *Covered Processors* | The processors inside the container that were executed during the MUnit test.
-| *Processors*         | The total amount of processors inside the container.
-| *Coverage*            | This field represents the percentage of the event processors inside the container that were executed in the MUnit test.
+| *Name*                | The name of the container
```

Comment:
I prefer to leave out the article (The) for easier scanning.

Reply:
OK! Thanks!